### PR TITLE
Specify credstash version

### PR DESCRIPTION
**Pull request by frankfu03**

### CODE REVIEW

File: travis/build.sh [hunk 1]
```diff
@@ -11,11 +11,11 @@ if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
 fi
 
 case "${TRAVIS_BRANCH}" in
-    "next")
+    "specify-credstash-version")
```
Comment (lbodor) [inline travis/build.sh]: Do we need this? I thought the above line was for only for testing.

File: aws/auscors-sitelogs-terraform/modules/lambda/create-ingest-text-site-log-package.sh [hunk 2]
```diff
@@ -10,7 +10,7 @@ if [ "$VIRTUAL_ENV" == "" ]; then
     (cd "$sourceDir" && pip install .)
 fi
 
-pip install requests lambda-packages credstash
+pip install requests lambda-packages credstash==1.16.1
```
Comment (lbodor) [inline aws/auscors-sitelogs-terraform/modules/lambda/create-ingest-text-site-log-package.sh]: Great.

File: travis/build.sh (hunk 1 above)
Comment (frankfu03) [reply travis/build.sh]: Yes, it is for testing and will be removed later. I am trying to deploy it to TEST site without merging.

File: aws/auscors-sitelogs-terraform/modules/lambda/create-ingest-text-site-log-package.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ if [ "$VIRTUAL_ENV" == "" ]; then
     (cd "$sourceDir" && pip install .)
 fi
 
-pip install requests lambda-packages credstash
+pip install requests lambda-packages cryptography==2.7 credstash==1.16.1
```
Comment (frankfu03) [inline aws/auscors-sitelogs-terraform/modules/lambda/create-ingest-text-site-log-package.sh]: credstash needs dependency: cryptography>=2.1. If credstash's version is 1.16.2, cryptography version 2.8 will be installed, but this cryptography version will caused two ImportErrors:
    - No module named asn1crypto.keys,
    - Cannot import name certificate_transparency.

Even if we downgrade credstash version to 1.16.1 or to 1.15.0, pip will still keep cryptography v2.8 as it satisfies the condition <= 2.1. So, explicitly install cryptography v2.7 will fix the errors.

Comment (lbodor) [reply aws/auscors-sitelogs-terraform/modules/lambda/create-ingest-text-site-log-package.sh]: Sounds good.

**Pull request merged**